projects/demo/src/modules/decorators/pure/pure-function.component.ts

```diff
@@ -5,12 +5,16 @@ import {changeDetection} from '../../../change-detection-strategy';
 @Component({
     selector: 'example-tui-pure-function',
     template: `
-        <tui-input [(ngModel)]="text"
-            >Type a text to start computing/tui-pure>
-            <div>Called times: {{ counter.count }}</div>
-            <div *ngIf="show">Result: {{ calculate(counter, text) | json }}</div>
-            <button tuiButton type="button" (click)="show = !show">Show/hide</button>
+        <tui-input [(ngModel)]="text">
+            Type a text to start computing/tui-pure
```

Comment:
I have no idea why we had "computing/tui-pure" here. Let's write smth like "Type a text to start computing" or "Type a text to start computing with tui-pure"

Reply:
Probably some auto-migration gone wrong :) I think "Type a text to start computing" is fine.

Reply:
Concluded like we can do it like @waterplea said